python/Framework.py

```diff
@@ -440,16 +413,13 @@ def applyMuonCorrection(self, type):
         if self.verbose:
             print("New muons collection: %r" % (self.__miniaod_muon_collection))
 
-        self.__muon_correction_done = True
-
+    @dep(before=("create", "eleReg"), performs=("correction", "eleReg"))
```

Comment:
as the previous flag was 'muonScale' I would prefer here "electronRegression", longer, but also more explicit

Reply:
ok, I'll change this one